CONTRIBUTING.md

```diff
@@ -90,6 +90,16 @@ We recommend the following commit structure in the following order:
 
 Follow [this guide](https://chris.beams.io/posts/git-commit/) to ensure you're writing good commit messages.
 
+## Creating a Release
+
+To create a production release of mbed-tools, perform the following steps:
+
+1. Check out the `master` branch and ensure it is up to date.
+1. Run `tox -e preprelease`, this will update the necessary files, create a commit and tag it with the new release version number.
```

Comment:
After running this myself, I have an untracked news file in my repo: news/20201106090030.feature

Inside, we have stuff like "Targets added: ...", "Targets removed: ...", "Targets modified: ...". I believe I can safely remove this news file as its contents have been properly added to the change log, but the preprelease process is leaving my branch dirty.

Reply:
Should all the news files be deleted in the "Prepare mbed-tools release" commit? I think we missed doing that.

Reply:
Towncrier should delete the news files when it creates the changelog. I'll have to look into why it's not doing that for the target database news file.

Reply:
I don't see it deleted any of the news files. I still have many newsfiles in my branch and tracked by git.

Reply:
Yes, that's because it failed quietly when it found the untracked one.